### PR TITLE
New download

### PR DESCRIPTION
**Pull request by Vital-jan**
Download, unzip & rename all file types - json, xml, zip using data.gov.ua API & unique dataset identifiers.

### CODE REVIEW

File: data_ocean/services/ruo_service.py [hunk 1]
```diff
@@ -5,11 +5,16 @@
...
     CHUNK_SIZE = 300
-    FILE_URL = ""
     LOCAL_FILE_NAME = "uo.xml"
-    
+    DATASET_ID = "1c7f3815-3259-45e0-bdf1-64dca07ddc10"
+
+    def rename_file (self, file):
+        new_filename = file
+        if (file.upper().find('UO') >= 0): new_filename = 'UO.xml'
+        if (file.upper().find('FOP') >= 0): new_filename = 'FOP.xml'
```
Comment (Iva-khar) [inline data_ocean/services/ruo_service.py:15]: In rfop_service names are 'uo.xml' & 'fop.xml' (lower case), why?

File: data_ocean/services/ratu_service.py [hunk 2]
```diff
@@ -29,8 +27,8 @@ class RatuConverter(Converter):
         'STREET_NAME': ''
     }
  
-    def rename (self, file):
-        new_filename = ""
+    def rename_file (self, file):
+        new_filename = file
         if (file.upper().find('ATU') >= 0): new_filename = 'ratu.xml'
         return new_filename
```
Comment (Iva-khar) [inline data_ocean/services/ratu_service.py:34]: Didn't it rename koatuu source file also?

Comment (Vital-jan) [reply data_ocean/services/ratu_service.py:34]: ++
May be this problem need to introduce a new algorythm...

File: data_ocean/services/ruo_service.py (hunk 1 above)
Comment (Vital-jan) [reply data_ocean/services/ruo_service.py:15]: Already fixed.

File: data_ocean/services/kved_service.py [hunk 3]
```diff
@@ -1,13 +1,12 @@
-import config
+# import config
```
Comment (criticus) [inline data_ocean/services/kved_service.py:1]: if you don't use it what's the reason to keep it?

File: data_ocean/services/ruo_service.py [hunk 4]
```diff
@@ -5,11 +5,16 @@
...
 class RuoConverter(Converter):
     
-    #paths for remote and local source files
     CHUNK_SIZE = 300
-    FILE_URL = ""
     LOCAL_FILE_NAME = "uo.xml"
-    
+    DATASET_ID = "1c7f3815-3259-45e0-bdf1-64dca07ddc10"
+
+    def rename_file (self, file):
```
Comment (criticus) [inline data_ocean/services/ruo_service.py:12]: DRY

File: data_ocean/services/kved_service.py (hunk 3 above)
Comment (Vital-jan) [reply data_ocean/services/kved_service.py:1]: Deleted.

File: data_ocean/services/ruo_service.py (hunk 4 above)
Comment (Vital-jan) [reply data_ocean/services/ruo_service.py:12]: What is dry?

**Pull request merged**